packages/create-video/src/init-v2.ts

```diff
@@ -0,0 +1,266 @@
+import execa from 'execa';
+import fs from 'fs-extra';
+import path from 'path';
+import stripAnsi from 'strip-ansi';
+import * as CreateDirectory from './create-directory';
+import {Log} from './log';
+import prompts, {selectAsync} from './prompts';
+
+type TEMPLATES = {
+	shortName: string;
+	name: string;
+	description: string;
+};
+
+const FEATURED_TEMPLATES: TEMPLATES[] = [
+	{
+		shortName: 'Helloworld',
+		name: 'template-helloworld',
+		description: 'a helloworld app with react logo',
+	},
+	{
+		shortName: 'Helloworld (Javascript)',
+		name: 'template-helloworld-javascript',
+		description: 'same as helloworld but with Javascript configuration',
+	},
+	{
+		shortName: 'Three Fiber template',
+		name: 'template-three',
+		description: 'Remotion + React Three Fiber Starter Template',
+	},
+];
+
+function padEnd(str: string, width: number): string {
+	// Pulled from commander for overriding
+	const len = Math.max(0, width - stripAnsi(str).length);
+	return str + Array(len + 1).join(' ');
+}
+
+function assertValidName(folderName: string) {
+	const validation = CreateDirectory.validateName(folderName);
+	if (typeof validation === 'string') {
+		throw new Error(
+			`Cannot create an app named ${chalk.red(
+				`"${folderName}"`
+			)}. ${validation}`
+		);
+	}
+}
+
+async function assertFolderEmptyAsync(
+	projectRoot: string,
+	folderName?: string
+) {
+	if (
+		!(await CreateDirectory.assertFolderEmptyAsync({
+			projectRoot,
+			folderName,
+			overwrite: false,
+		}))
+	) {
+		const message = 'Try using a new directory name, or moving these files.';
+		Log.newLine();
+		Log.info(message);
+		Log.newLine();
+		throw new Error(message);
+	}
+}
+
+const shouldUseYarn = (): boolean => {
+	return Boolean(
+		process.env.npm_execpath?.includes('yarn.js') ||
+			process.env.npm_config_user_agent?.includes('yarn')
+	);
+};
+
+const isGitExecutableAvailable = async () => {
+	try {
+		await execa('git', ['--version']);
+		return true;
+	} catch (e) {
+		if (e.errno === 'ENOENT') {
+			Log.warn('Unable to find `git` command. `git` not in PATH.');
+			return false;
+		}
+	}
+};
+
+const initGitRepoAsync = async (
+	root: string,
+	flags: {silent: boolean; commit: boolean} = {silent: false, commit: true}
+) => {
+	// let's see if we're in a git tree
+	try {
+		await execa('git', ['rev-parse', '--is-inside-work-tree'], {
+			cwd: root,
+		});
+		!flags.silent &&
+			Log.info(
+				'New project is already inside of a git repo, skipping git init.'
+			);
+	} catch (e) {
+		if (e.errno === 'ENOENT') {
+			!flags.silent &&
+				Log.warn('Unable to initialize git repo. `git` not in PATH.');
+			return false;
+		}
+	}
+
+	// not in git tree, so let's init
+	try {
+		await execa('git', ['init'], {cwd: root});
+		!flags.silent && Log.info('Initialized a git repository.');
+
+		if (flags.commit) {
+			await execa('git', ['add', '--all'], {cwd: root, stdio: 'ignore'});
+			await execa('git', ['commit', '-m', 'Create a new Remotion app'], {
```

Comment:
Let's not use "app" as the term for a project but "video"

Reply:
got it :fire: